server/src/main/java/org/elasticsearch/search/aggregations/metrics/MedianAbsoluteDeviationAggregationBuilder.java

```diff
@@ -63,9 +63,13 @@ public MedianAbsoluteDeviationAggregationBuilder(String name) {
     public MedianAbsoluteDeviationAggregationBuilder(StreamInput in) throws IOException {
         super(in);
         compression = in.readDouble();
-        executionHint = in.getTransportVersion().onOrAfter(TransportVersion.V_8_500_014)
-            ? TDigestExecutionHint.readFrom(in)
-            : TDigestExecutionHint.HIGH_ACCURACY;
+        if (in.getTransportVersion().onOrAfter(TransportVersion.V_8_500_017)) {
+            if (in.readBoolean()) {
```

Comment:
I think `readOptionalWriteable(...)` can be used here? This turns this into a one liner.

Reply:
Oh nice, missed this version. Done.